src/main/java/com/code_intelligence/jazzer/agent/RuntimeInstrumentor.kt

```diff
@@ -181,7 +181,10 @@ class RuntimeInstrumentor(
             .acceptClasses(className)
             .scan()
             .use {
-                it.getClassInfo(className).resource.load()
+                it.getClassInfo(className)?.resource?.load() ?: run {
```

Comment:
Could you mention this change in the commit message or split it out into a separate commit?

Reply:
done.